Enterprise-CMCS/macpro-mdct-mcr
# MDCT-2235: MLR Page 2 Filled State

## Summary

### Description

This creates the filled state and adds a status for MLR reports.

Some notes:
- Currently, the status will stay at the default (not started) and present a warning
- The `Enter MLR` button has no functionality (yet, out of scope for this ticket)

New functionalities included here:
- Editing an MLR program
- Deleting an MLR program

### Related ticket

MDCT-2235 and MDCT-2223

### How to test

- Log in as a state user
- Create a new MLR report
- Navigate to `MLR Reporting`
- Add new program reporting information rows
- Edit that program
- Delete the program

Make sure that the MLR reports counter is going up correctly as you add new ones and that it goes down as you delete 'em.

### Important updates

N/A

### Author checklist
